concent-builder/configure-and-run-tests.yml

```diff
@@ -0,0 +1,55 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Copy concent and virtualmachine for deployment build
+          command: cp                                                                  \
+              --recursive                                                              \
+              "{{ build_dir }}/concent-deployment/containers/build/{{ item.path }}"    \
+              "{{ jenkins_home_dir }}/{{ item.name }}"
+          with_items:
+            - { path: "repositories/concent", name: "concent" }
+            - { path: "virtualenv", name: "virtualenv" }
+
+        - name: Copy local_settings.py file
+          copy:
+            src:        "{{ jenkins_home_dir }}/concent-deployment/concent-builder/jenkins-configuration-files/local_settings.py"
+            dest:       "{{ jenkins_home_dir }}/concent/concent_api/concent_api/settings/local_settings.py"
+            owner:      jenkins
+            group:      jenkins
+            remote_src: yes
+
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Wait for geth synchronization
+          command: ./wait-until-ready.sh geth 600
+          args:
+            chdir:  "{{ build_dir }}/concent-deployment/kubernetes/build/"
```

Comment:
Waiting for geth to get synchronized should be a separate step in the pipeline. We want to be able to see that it's the synchronization that failed and not the tests themselves.

Reply:
Do we still want to wait for geth? After we merged pull request with liveness and readiness probe for geth, the deployment process wait 5 minutes for geth synchronous and return timeout when time exceed.

Reply:
In that case you can ignore it here. We'll come back to this in the future if it turns out that a separate waiting step would be more convenient.